core/arch/arm/kernel/spin_lock_a64.S

```diff
@@ -55,35 +55,34 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 #include <kernel/tz_proc.h>
-
 #include <asm.S>
 
-/* void cpu_spin_lock(unsigned int *lock); */
-FUNC cpu_spin_lock , :
+/* void __cpu_spin_lock(unsigned int *lock); */
+FUNC __cpu_spin_lock , :
 	mov	w2, #SPINLOCK_LOCK
 	sevl
-l1:	wfe
-l2:	ldaxr	w1, [x0]
-	cbnz	w1, l1
+1:	wfe
```

Comment:
This function is copied from ARM-TF `lib/locks/exclusive/aarch64/spinlock.S`, please keep it as close as possible to the original.